tests/test_search_issue_api.py

```diff
@@ -0,0 +1,25 @@
+import allure
+
+from src import rest_client
+
+
+@allure.tag("API")
+def test_get_valid_issue_api(create_issue, delete_issue):
+    issue_id = create_issue.json()["key"]
+    response = rest_client.find_issue(issue_id=issue_id)
+    print(response.text)
+    assert response.status_code == 200
```

Comment:
Please, add check for the Content Type (JSON, XML, Text) and check that response contains anything that would confirm that response is valid.

Reply:
@archick12 Added necessary checks.